packages/kit/src/module/install.ts

```diff
@@ -35,6 +37,17 @@ export async function installModule (moduleToInstall: string | NuxtModule, _inli
 
 // --- Internal ---
 
+export const normaliseModuleTranspilePath = (p: string) => {
```

Comment:
probably we want this to be US English. 🥲

```suggestion
export const normalizeModuleTranspilePath = (p: string) => {
```

Reply:
haha yup, always catches me out

Reply:
(it'll have to be the other files too)